packages/core-data/src/selectors.ts

```diff
@@ -69,6 +77,7 @@ interface EntityState< EntityRecord extends ET.EntityRecord > {
 	>;
 	deleting: Record< string, Partial< { pending: boolean; error: Error } > >;
 	queriedData: QueriedData;
+	revisions: RevisionsQueriedData;
```

Comment:
I think that shape of reducer is perfect. It matches exactly why I had in mind, what I didn't understand in the reducer is why we were not able to just reuse the regular queried state reducer.

Reply:
Thanks, I'll look into it and try out your idea of updating the action to use the queried state reducer as a "helper"